Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -1854,6 +1854,7 @@ public void EvaluateBlockAction()
             Assert.Equal(
                 (Integer)2,
                 (Integer)blockActionEvaluation.OutputStates.GetState(miner));
+            Assert.True(blockActionEvaluation.InputContext.BlockAction);
```

Comment:
Could we have a more test case where this property turns out to be `false`?

Reply:
https://github.com/planetarium/libplanet/pull/1143/commits/f46d693c43f9f1d76f33b1f36489408580812bb0 I add it